Apps/CommonData/src/Utils/DateFormatter.cs

```diff
@@ -23,6 +23,8 @@ namespace HealthGateway.Common.Data.Utils;
 /// </summary>
 public static class DateFormatter
 {
+    private static readonly TimeZoneInfo PstZone = TimeZoneInfo.FindSystemTimeZoneById("Pacific Standard Time");
```

Comment:
Unfortunately, this is not going to work in a linux container, you'll need to do something similar to [[bcgov/embc-ess-mod@master/shared/src/EMBC.Utilities/Extensions/DateTimeEx.cs](https://github.com/bcgov/embc-ess-mod/blob/master/shared/src/EMBC.Utilities/Extensions/DateTimeEx.cs)](https://github.com/bcgov/embc-ess-mod/blob/master/shared/src/EMBC.Utilities/Extensions/DateTimeEx.cs#L17)

Reply:
We already have code that does this in JobScheduler and it can be moved into CommonData.  JobScheduler.Utils.ScheduleHelper GetLocalTimeZone

https://github.com/bcgov/healthgateway/blob/dev/Apps/JobScheduler/src/Utils/SchedulerHelper.cs

Reply:
@sslaws moved GetLocalTimeZone to Common.Data.DateFormatter.cs.  Spoke with @ytqsl regarding hardcoding timezone ids instead of accessing via configuration as those values will most likely never change.